CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 2025-05-20, Version 22.7.0 (Stable), @Rhodine-orleans-lindsay
+### Added
+- Service paused funtionality that allows for services to redirect to a 'Service Unavailable' when there is a need to pause a service:
```

Comment:
typo with `funtionality` but I'll fix it
and just minor points about the description